python3/libexec/nbd_client_manager.py

```diff
@@ -1,52 +1,66 @@
-#!/usr/bin/python
+#!/usr/bin/python3
 
 """
 Provides functions and a CLI for safely connecting to and disconnecting from
 NBD devices.
 """
 
 import argparse
+import fcntl
+import json
 import logging
 import logging.handlers
 import os
+import re
 import subprocess
 import time
-import fcntl
-import json
-import re
 from datetime import datetime, timedelta
 
-
 LOGGER = logging.getLogger("nbd_client_manager")
 LOGGER.setLevel(logging.DEBUG)
 
-LOCK_FILE = '/var/run/nonpersistent/nbd_client_manager'
+LOCK_FILE = "/var/run/nonpersistent/nbd_client_manager"
 
 # Don't wait more than 10 minutes for the NBD device
 MAX_DEVICE_WAIT_MINUTES = 10
 
 
+class NotGetNbdNumber(Exception):
+    """
+    The NBD device should be in this format: nbd{0-100}
```

Comment:
The following line suggests there may be 1000 :(
`for device_no in range(0, 1000):`